src/documents/tests/test_management_consumer.py

```diff
@@ -411,3 +412,101 @@ def test_consume_file_with_path_tags(self):
     )
     def test_consume_file_with_path_tags_polling(self):
         self.test_consume_file_with_path_tags()
+
+
+class TestConsumerOwner(DirectoriesMixin, ConsumerThreadMixin, TransactionTestCase):
```

Comment:
Please add the given/when/then convention to provide some context for what each of these is trying to accomplish